Formula/kube-score.rb

```diff
@@ -0,0 +1,35 @@
+class KubeScore < Formula
+  desc "Kubernetes object analysis recommendations for improved reliability and security"
+  homepage "https://kube-score.com"
+  url "https://github.com/zegl/kube-score/archive/refs/tags/v1.13.0.tar.gz"
+  sha256 "a09198f9a49c0008f9a34b418d85d3eb1299d0f04077e423eff30f1cad8abd43"
```

Comment:
let's change to a git checkout so we can access the commit hash below
```suggestion
  url "https://github.com/zegl/kube-score.git",
      tag:      "v1.13.0",
      revision: "d1ad91defc1a5814f7c7395ed64cd7039d259158"
```